watchtower/compose.yml

```
@@ -0,0 +1,8 @@
+services:
+  watchtower:
+    image: containrrr/watchtower
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+    command: 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)